app/services/katello/pulp3/repository/apt.rb

```diff
@@ -0,0 +1,81 @@
+require 'pulp_deb_client'
+
+module Katello
+  module Pulp3
+    class Repository
+      class Apt < ::Katello::Pulp3::Repository
+        def remote_options
+          deb_remote_options = {
+            distributions: root.deb_releases
+          }
+          # TODO: need to define how to set these fields to empty
```

Comment:
can you explain a bit about this TODO?

Reply:
In the version of pulp_deb that I used for testing, there was no process for emptying these fields.
That meant that supplying an empty string resulted in nothing being synched (or not being accepted by pulp) instead of requesting all available releases/components/architectures.

I will check again with with the pulp_deb team once I get my testing forklift up again using the new rpm for pulp_deb plugin.

Reply:
This should be long fixed. Can we resolve this thread?